checkov/common/bridgecrew/platform_integration.py

```diff
@@ -940,7 +940,9 @@ def persist_repo_id(self, args: argparse.Namespace) -> str:
     def get_repository(self, args: argparse.Namespace) -> str:
         if CI_METADATA_EXTRACTOR.from_branch:
             return CI_METADATA_EXTRACTOR.from_branch
-        basename = 'unnamed_repo' if path.basename(args.directory[0]) == '.' else path.basename(args.directory[0])
+        arg_dir = args.directory[0]
+        arg_dir.rstrip(os.path.sep)  # If directory ends with /, remove it. Does not remove any other character!!
```

Comment:
if you want to can also merge those lines into one

Reply:
Then the line is too long and lint complains lol